code/modules/mob/living/carbon/life.dm

```diff
@@ -21,9 +21,9 @@
 		if(.) //not dead
 			handle_blood()
 
-		if(isLivingSSD())//if you're disconnected, you're going to sleep
-			if(AmountSleeping() < 20)
-				AdjustSleeping(20)//adjust every 10 seconds
+//		if(isLivingSSD())//if you're disconnected, you're going to sleep
+//			if(AmountSleeping() < 20)
+//				AdjustSleeping(20)//adjust every 10 seconds
```

Comment:
what's up with this?

Reply:
argh I don't know why did it didn't revert that, it was just for testing